book/.vitepress/config.mts

```diff
@@ -35,6 +35,7 @@ export default defineConfig({
             { text: 'Getting Started', link: 'getting-started' },
             { text: 'Configuring', link: 'configuring' },
             { text: 'Initializing', link: 'initializing' },
+            { text: 'Hardware Requirements', link: 'hardware-requirements' },
```

Comment:
Not the right place I think. Just add it to the top of the getting started guide.